released/packages/coq-libhyps/coq-libhyps.1.0.0/opam

```diff
@@ -38,6 +38,6 @@ premisses, but define your own manipulation on (groups of) hypothesis.
 "
 
 url {
-    http: "https://github.com/Matafou/LibHyps/archive/libhyps-1.0.0.tar.gz"
+    http: "https://github.com/Matafou/LibHyps/archive/libhyps-1.0.1.tar.gz"
```

Comment:
This is very strange to me. The old release is still there, so I think there are two options here:

1. Make a new opam package for 1.0.1, independent of 1.0.0.
2. Simply rename this package/directory to `coq-libhyps-1.0.1` (if release 1.0.0 is no longer useful)

We want to avoid a situation where opam package versions are not in synch with releases.

Reply:
you are right. Something is wrong.
1.0.0 is no longer useful, even for previous versions of coq.
I will rename the directory and make another PR.

Reply:
It's up to you to create another PR, but another option is also to just force-push the directory change your branch for the current PR. Either is fine.

Reply:
yeah but this one was created from github directly, with a dummy name. I prefer recreating the branch.
Thanks!